An error occurred while executing the following cell:
------------------
try:
    import cv2
except:
    %pip install cv2
    import cv2
import jax

try:
    import augmax
except:
    %pip install git+https://github.com/khdlr/augmax.git -q
    import augmax
from jax import numpy as jnp
from jax import random as jrand

try:
    from skimage.util import montage
except:
    %pip install skimage
    from skimage.util import montage

try:
    import tensorflow_datasets as tfds
except:
    %pip install tensorflow_datasets
    import tensorflow_datasets as tfds

import matplotlib.pyplot as plt

%matplotlib inline
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_1457/3981733033.py in <module>
      1 try:
----> 2     import cv2
      3 except:

ModuleNotFoundError: No module named 'cv2'

During handling of the above exception, another exception occurred:

ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_1457/3981733033.py in <module>
      3 except:
      4     get_ipython().run_line_magic('pip', 'install cv2')
----> 5     import cv2
      6 import jax
      7 

ModuleNotFoundError: No module named 'cv2'
ModuleNotFoundError: No module named 'cv2'
